source/httpfs/http_filesystem_test.go

```diff
@@ -1,3 +1,4 @@
+//go:generate esc -o escfs/escfs.go -pkg escfs -prefix escfs/ escfs/
```

Comment:
It is referenced only for code generation. It is not used for build or test. This is just a helper for anyone wanting to change tests that use embedded file system. Anyone who does not change ESC test files will not be using this nor would it affect their build.

Reply:
I'd like to avoid code-gen if possible. It seems like the same testing needs can be met by `afero`. https://github.com/spf13/afero#using-afero-for-testing